tests/apidocs/endpoints/events/test_project_issues.py

```diff
@@ -4,7 +4,7 @@
 from sentry.testutils.silo import region_silo_test
 
 
-@region_silo_test
+@region_silo_test(stable=True)
```

Comment:
Adding some more hybrid cloud stable tests -- this causes the decorated suite to run once "as is", and a second time with the hybrid cloud logic in place.  In this case, it required a small adjustment to the hybrid cloud authorization flow.  Has no impact on production (currently) but keeps our test coverage up and expands the various features we support in region mode.